src/engine/Server.cpp

```diff
@@ -306,7 +307,7 @@ Awaitable<void> Server::process(
   std::optional<http::response<http::string_body>> response;
 
   // Execute commands (URL parameter with key "cmd").
-  auto logCommand = [](const std::optional<std::string>& cmd,
+  auto logCommand = [](const std::optional<std::string_view>& cmd,
```

Comment:
While you are at it, you can use such a small PR to do some refactoring:
Extract this block (the one that handles all the commands)
as well as the one above (that checks the access token) into separate functions, s.t. this becomes
```
bool accessTokenOk = checkAccessToken(checkParameter("access-token", std::nullopt));
std::optional<...> response = handleCommands(...).
```